orc8r/cloud/go/sqorc/builder.go

```diff
@@ -318,6 +318,26 @@ func ClearStatementCacheLogOnError(cache *squirrel.StmtCache, callsite string) {
 	}
 }
 
+// FmtConflictUpdateTarget generates the string used to refer to target value
+// used in an on-conflict upsert operation; the specific syntax depends on the
+// sql dialect.
+// e.g. for mariaDB, "f1=t2.f1"; for postgres, "f1=excluded.f1"
+func FmtConflictUpdateTarget(tableName string, colName string) string {
+	dialect, envFound := os.LookupEnv("SQL_DIALECT")
```

Comment:
"dialect, ok" is standard and concise enough for me

Reply:
"dialect, envFound" is used in the `GetSqlBuilder` function in the same file; do you suggest I prioritize consistency or conciseness? @hcgatewood

Reply:
Ah I see 🙂 In that case either is fine. Just make them both the same